tools/ci-build/changelogger/src/render.rs

```diff
@@ -62,11 +59,12 @@ pub struct RenderArgs {
     /// Source changelog entries to render
     #[clap(long, action, required(true))]
     pub source: Vec<PathBuf>,
-    /// Which source to overwrite with an empty changelog template
-    #[clap(long, action)]
-    pub source_to_truncate: PathBuf,
+    /// Where to output the rendered changelog entries
     #[clap(long, action)]
     pub changelog_output: PathBuf,
+    /// Optional directory path to the changelog directory to empty, leaving only `.example` in it
+    #[clap(long, action)]
+    pub source_to_truncate: Option<PathBuf>,
```

Comment:
Yes we do. `changelogger` is also used for rendering release notes in the `aws-sdk-rust` repository; there isn't a `.changelog` directory there. Since there isn't anything to truncate, the `render` subcommand won't specify `--source-to-truncate` when rendering release notes in the `aws-sdk-rust` repository.